packages/dappmanager/src/initializeDb.ts

```diff
@@ -211,6 +211,7 @@ export default async function initializeDb(): Promise<void> {
   // - Updates the domain: db.domain.set(domain);
   dyndns.generateKeys(); // Auto-checks if keys are already generated
 
+  // TODO: Move this functionality to Bind v2 package
```

Comment:
Here we are mapping the resolution of the `dyndns` to the private IP, instead of the public one, to improve UX. This should be done in the bind package instead. We can leave this here only until this is implemented